### PR TITLE
add jquery-nstslider v1.0.13 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/lokku/jquery-nstslider
**Watch 16
  Star 211
 Fork 39**
@Piicksarn could u help me check this PR for #6284 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/jquery-nstslider/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "jquery-nstslider",
+  "filename": "jquery.nstSlider.min.js",
+  "description": "Fully customizable with CSS, Single/Double handles, Touch-enabled, IE 7+ Compatibility, Custom Digit Rounding, Non linear step increments!",
+  "keywords": [
+    "Single/Double handles",
+    "Touch-enabled",
+    "IE 7+ Compatibility",
+    "Custom Digit Rounding"
+  ],
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-nstslider/package.json]: These are not "keywords", you only used a complex string, think about what does keywords mean, and see: https://github.com/lokku/jquery-nstslider/blob/master/nstSlider.jquery.json

**Comment by maruilian11** [comment]
@Piicksarn could your help me check this PR?

**Comment by PeterDaveHello** [comment]
As I said there is keywords from upstream, why don't you use it?

**Comment by maruilian11** [comment]
@Piicksarn could you help me check this Pr?

**Comment by Piicksarn** [comment]
It's fine!
@PeterDaveHello, would you please have a look at the pr?
Thanks.

**Pull request merged**